ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1640,6 +1640,7 @@ private class JdbcLedgerDao(
         |truncate contract_keys cascade;
         |truncate configuration_entries cascade;
         |truncate package_entries cascade;
+        |truncate participant_command_completions;
```

Comment:
Everything else cascades; why not this one?

Reply:
Because I like dangling indexes. :smile: Good catch, thanks, I'll fix this.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4681/commits/1f57a422e18c1ddb39b6591e2860812cbfd9a1d9.